src/display/api.js

```diff
@@ -884,6 +884,15 @@ class PDFDocumentProxy {
   getFieldObjects() {
     return this._transport.getFieldObjects();
   }
+
+  /**
```

Comment:
Let's replace this block with the following block that contains more information about the `null` case, which makes it consistent with the recently changed https://github.com/mozilla/pdf.js/pull/12483/files#diff-082d6b37ad01db7ac97cc07c6ddb0dc52040484c5ef91b110b072f50144d9f39R881-R883:

```javascript
  /**
   * @returns {Promise<Array<string> | null>} A promise that is resolved with an
   *   {Array<string>} containing IDs of annotations that have a calculation
   *   action, or `null` when no such annotations are present in the PDF file.
   */
```